docs/src/overview/readme.stories.mdx

```diff
@@ -131,9 +131,11 @@ import {Icon} from 'dooboo-ui';
 ## List of independent components in `@dooboo-ui/*`
 
 > Below elements are not included in `dooboo-ui` package since it depends on other packages that may overload its package.
+> We are the packages modules are maintained with [lerna](https://lerna.js.org/) to maintain the mono repo. For more information, please read [dooboo-ui-strategy](https://github.com/dooboolab/dooboo-ui/pull/49)
```

Comment:
I thinking changing words to `Here are separate packages that are maintained with Lerna for the mono repo.` would look better.

Reply:
Thanks! I'll edit it and upload it again! 😁